src/Widgets/FormStepContainerWidget/FormStepContainerWidgetEditingConfig.ts

```diff
@@ -269,12 +271,13 @@ function getNavigationProperties(obj) {
     return MultiStepNavigationProps;
   }
 }
+
 /**
  * Retrieves the properties for the review tab
  * @param {*} obj
  * @returns
  */
-function getReviewProperties(obj) {
+function getReviewProperties(obj: Scrivito.Widget): string[] | any[] {
```

Comment:
I'm not sure about the responsibilities of the getReviewProperties function. However if it's intended to handle a particular widget type, you might consider using ts generics, such as Scrivito.Widget<SpecificWidgetDefinition>. I can't guarantee this will work. If would you wouldn't need to use "as string". Also, if the parameter is indeed a widget, naming it `widget` would be more descriptive and appropriate.